docs/demo.ascii

```diff
@@ -0,0 +1,101 @@
+{"version": 2, "width": 202, "height": 55, "timestamp": 1703077313, "env": {"SHELL": "/bin/bash", "TERM": "xterm-256color"}}
```

Comment:
Was this created with the auto-type thingy? There's a type correction in it now :smile:

Reply:
Not sure what this is about...?

Reply:
The deterministic demo creation tool I shared with you earlier. There's currently a typo in the demo animation. If we're going to add demo animations, we should at least make sure they look clean and errorless.

Reply:
Oh, you mean, the part of the animation where I correct something. Okay, well, I thought it was better to leave it in, because it gives you more of "real life experience". But I can change it if you want.